postgresql/installingpostgresqlonibmi.md

```diff
@@ -0,0 +1,167 @@
+```
+--------------------------------------------------------------------------------
+                        Work with IPv4 Connection Status                   
+                                                             System:   SYS1
+ Type options, press Enter.                                                
+   3=Enable debug   4=End   5=Display details   6=Disable debug            
+   8=Display jobs                                                          
+                                                                           
+      Remote           Remote     Local                                    
+ Opt  Address          Port       Port       Idle Time  State              
+      *                *          5432       000:05:22  Listen             
+--------------------------------------------------------------------------------
+```
+
+Allow remote access to server. You can change this by editing ***/postgres/pg_hba.conf*** 
+
+Add following entry to the pg_hba.conf file:
+```
+host   all   all   0.0.0.0/0     password
+```
+Note: This will enable password checking. You'll need to set a new password for the ***postgres*** user on the Postgresql server. See below.
+
+Save pg_hba.conf. 
+
+Stop and then start Postgresql server as described previously. 
+
+From shell command line, create demo postgres database using the following command line command:
+```
+createdb ibmidemo
+```
+
+The psql command utility can be used to provide permissions and do other server related maintenance. 
+
+For now we will just allow access for the postgres user to our new database as an example.
+We will also set a password for the postgres user. This sample uses 'postgres2020' for the password. 
+You should use a more secure password. 
+***NOTE: You should also review the Postgres site for appropriate Postgres security measures.***
+
+Start psql utility
+```
+psql 
+```
+
+Type the following sql and press Enter to set database access for postgres database user:
+```
+grant all privileges on database ibmidemo to postgres;
+```
+
+Type the following sql and press Enter to set the database server password for the postgres database user:
+```
+alter user postgres with password 'postgres2020';
+```
+
+Type: ***quit*** and press enter to exit the psql utility.
+
+
+Use Heidi, DBeaver or other Postgresql client to connect to Postgres database. 
+```
+Host: IBMi host name or IP
+Port: 5432
+User: postgres
+Password: postgres2020
+Database: ibmidemo
+```
+
+If desired, change the port that Postgresql server listens on to something other than 5432.
+
+use nano editor, vim or other editor to edit ***/postgres/postgresql.conf*** file 
+
+```
+Change port number. 
+Ex: port = 60432 
+```
+
+save postgresql.conf
+
+Stop and restart Postgres server
+
+Now refer to standard Postgresql documentation as needed. 
+
+# Links
+
+Postgresql Site
+
+https://www.postgresql.org
```

Comment:
@richardschoen 

We should link to the postgresql 12 docs https://www.postgresql.org/docs/12/index.html.